services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -482,7 +482,17 @@ module.exports = {
       if (section) return section;
 
       // Current section does not match site, load alternate.
-      // @todo This should eventually account for secondary sites/sections. For now, load home.
+      const sectionQuery = getAsArray(content, 'sectionQuery');
```

Comment:
This can be accomplished by returning _one_ section, not all of them. No need to query and return all possible sections when you're only returning one of them. :)

Instead, change your to exclude the current siteId context, include where the schedule section Ids exist, and where the alias _is not_ home

Reply:
Something like:

```js
const currentSiteSectionQuery = {
  ...formatStatus(status),
  'site.$id': { $ne: siteId },
  _id: { $in: sectionQuery.map(schedule => schedule.sectionId) },
  alias: { $ne: 'home' },
};
```

Also, the if should also require that `siteId` be present. If no site context is provided then there's no way to know what to return

Reply:
> This can be accomplished by returning _one_ section, not all of them. No need to query all possible sections when you're only returning one of them. :)
> 
> Instead, change your query to exclude the current siteId context, include where the schedule section Ids exist, and where the alias _is not_ home

The current siteId context is correct at this point (it _should_ be included, not excluded), but agreed on `findOne` instead of `find` and the alias

Reply:
Yep, already fixed the siteId part

Reply:
```js
const currentSiteSectionQuery = {
  ...formatStatus(status),
  'site.$id': siteId,
  _id: { $in: sectionQuery.map(schedule => schedule.sectionId) },
  alias: { $ne: 'home' },
};
```